### PR TITLE
Feature/JM-6420- As a jury officer i want to generate a unpaid attendance report (detailed)

### PR DESCRIPTION
**Pull request by hari-vajja**
### JIRA link (if applicable) ###

https://centralgovernmentcgi.atlassian.net/browse/JM-6420

### Change description ###
Added functionality to generate an unpaid attendance report

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: client/templates/reporting/standard-reports/bespoke-report-body/unpaid-attendance-detailed.njk [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <td class="govuk-table__cell" colspan="6"><p class="govuk-body-l govuk-!-margin-0">{{ item.date | dateFilter("YYYY,MM,DD", "dddd D MMMM YYYY") }}</p></td>
+    </tr>
+    {% for row in item.values %}
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell mod-!-width-one-sixth"><a href="{{ url('juror-record.overview.get', { jurorNumber: row.jurorNumber }) }}" class="govuk-link">{{row.jurorNumber}} </a></td>
+      <td class="govuk-table__cell mod-!-width-one-sixth">{{ row.firstName }}</td>
+      <td class="govuk-table__cell mod-!-width-one-sixth">{{ row.lastName }}</td>
+      <td class="govuk-table__cell mod-!-width-one-sixth"><a href="{{ url('reports.financial-audit.get', { auditNumber: row.auditNumber }) }}" target="_blank">
+              {{ row.auditNumber }}
+            </a></td>
```
Comment (callum-r-young) [inline client/templates/reporting/standard-reports/bespoke-report-body/unpaid-attendance-detailed.njk:25]: So for this link, if the audit number begins with 'P' it should link to the 'pool-attendances-audit' report and if it is 'J' then should link to 'jury-attendance-audit' report

File: server/routes/reporting/standard-report/definitions.js [hunk 2]
```diff
@@ -1343,6 +1343,24 @@
           totals: true,
         }
       },
+      'unpaid-attendance-detailed': {
+        title: 'Unpaid attendance report (detailed)',
+        apiKey: 'UnpaidAttendanceReportDetailedReport',
+        search: 'dateRange',
+        printLandscape: true,
```
Comment (callum-r-young) [inline server/routes/reporting/standard-report/definitions.js:1350]: On Figma the document is printed Portrait but you have Landscape?

**Review by callum-r-young — CHANGES_REQUESTED** [review]

File: client/templates/reporting/standard-reports/bespoke-report-body/unpaid-attendance-detailed.njk (hunk 1 above)
Comment (hari-vajja) [reply client/templates/reporting/standard-reports/bespoke-report-body/unpaid-attendance-detailed.njk]: Done

File: server/routes/reporting/standard-report/definitions.js (hunk 2 above)
Comment (hari-vajja) [reply server/routes/reporting/standard-report/definitions.js]: Changed

**Review by callum-r-young — APPROVED** [review]

**Pull request merged**